packages/@webex/plugin-meetings/test/unit/spec/meeting/index.js

```diff
@@ -1623,438 +1655,421 @@ describe('plugin-meetings', () => {
-          const simulateRoapOffer = async () => {
-            const roapListener = getRoapListener();
+        // simulates a Roap offer being generated by the RoapMediaConnection
+        const simulateRoapOffer = async () => {
+          const roapListener = getRoapListener();
 
-            await roapListener({roapMessage: roapOfferMessage});
-            await stableState();
-          };
+          await roapListener({roapMessage: roapOfferMessage});
+          await stableState();
+        };
 
-          const checkSdpOfferSent = ({audioMuted, videoMuted}) => {
-            const {sdp, seq, tieBreaker} = roapOfferMessage;
-
-            assert.calledWith(locusMediaRequestStub, {
-              method: 'PUT',
-              uri: `${meeting.selfUrl}/media`,
-              body: {
-                device: {
-                  url: meeting.deviceUrl,
-                  deviceType: meeting.config.deviceType,
-                  regionCode: 'EU',
-                  countryCode: 'UK',
-                },
-                correlationId: meeting.correlationId,
-                localMedias: [
-                  {
-                    localSdp: `{"audioMuted":${audioMuted},"videoMuted":${videoMuted},"roapMessage":{"messageType":"OFFER","sdps":["${sdp}"],"version":"2","seq":"${seq}","tieBreaker":"${tieBreaker}"}}`,
-                    mediaId: 'fake media id',
-                  },
-                ],
-                clientMediaPreferences: {
-                  preferTranscoding: !meeting.isMultistream,
-                  joinCookie: undefined,
-                  ipver: 0,
-                },
-              },
-            });
-          };
+        const checkSdpOfferSent = ({audioMuted, videoMuted}) => {
+          const {sdp, seq, tieBreaker} = roapOfferMessage;
 
-          const checkLocalMuteSentToLocus = ({audioMuted, videoMuted}) => {
-            assert.calledWith(locusMediaRequestStub, {
-              method: 'PUT',
-              uri: `${meeting.selfUrl}/media`,
-              body: {
-                device: {
-                  url: meeting.deviceUrl,
-                  deviceType: meeting.config.deviceType,
-                  regionCode: 'EU',
-                  countryCode: 'UK',
-                },
-                correlationId: meeting.correlationId,
-                localMedias: [
-                  {
-                    localSdp: `{"audioMuted":${audioMuted},"videoMuted":${videoMuted}}`,
-                    mediaId: 'fake media id',
-                  },
-                ],
-                clientMediaPreferences: {
-                  preferTranscoding: !meeting.isMultistream,
-                  ipver: 0,
+          assert.calledWith(locusMediaRequestStub, {
+            method: 'PUT',
+            uri: `${meeting.selfUrl}/media`,
+            body: {
+              device: {
+                url: meeting.deviceUrl,
+                deviceType: meeting.config.deviceType,
+                regionCode: undefined,
```

Comment:
why are regionCode and countryCode expected values undefined now? (here and in lines 1845-1846)

Reply:
Done @marcin-bazyl